CHANGES.md

```diff
@@ -6,5 +6,6 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### What's new
 
+- Support large uploads for Admin Router service endpoint. (DCOS-52768)
```

Comment:
Do we not want to backport this to 1.12.x, 1.13.x? Then this should not be here.

Reply:
As far as I understand the underlying COPS ticket this is actually something we want to change in 1.12.x, and 1.13.x, too. In that case this changelog entry should go into the corresponding point release sections. By the time 1.14.0 will be released this is then not actually new, and this entry here should not be there.

Reply:
What if they upgrade from 1.13.0 to 1.14.0?

Reply:
Is the answer to 

> Do we want to have this change in 1.12.x, 1.13.x?

"yes"?

> What if they upgrade from 1.13.0 to 1.14.0?

We want to focus on telling them about breaking changes, and about the generally new things in this major release.

The "### What's new" section is meant to capture things that are *new in this major release*.

I hope this provides a somewhat satisfying answer:

https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines#major-releases